projects/perception/face_recognition/demos/inference_demo.py

```diff
@@ -33,8 +33,8 @@
 
     recognizer.download(path=".", mode="test_data")
     recognizer.fit_reference(path=join(".", "test_data", "images"), save_path=".")
-    image_path = join(".", "test_data", "images", "Mr. Bean", "1.jpg")
-    img = cv2.imread(image_path)
```

Comment:
We should probably adapt the examples provided in the documentation accordingly too, but isn't critical

Reply:
I think I updated any `cv2.imread()` instances. I also fixed some old `OpenDR` import statements (this PR is now a bit messy...).